irohad/torii/impl/command_service_transport_grpc.hpp

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef TORII_COMMAND_SERVICE_TRANSPORT_GRPC_HPP
+#define TORII_COMMAND_SERVICE_TRANSPORT_GRPC_HPP
+
+#include "torii/command_service.hpp"
+
+#include <chrono>
+
+#include "endpoint.grpc.pb.h"
+#include "endpoint.pb.h"
+#include "interfaces/iroha_internal/tx_status_factory.hpp"
+#include "logger/logger.hpp"
+#include "torii/status_bus.hpp"
+
+namespace torii {
+  class CommandServiceTransportGrpc
+      : public iroha::protocol::CommandService::Service {
+   public:
+    /**
+     * Creates a new instance of CommandServiceTransportGrpc
+     * @param command_service - to delegate logic work
+     * @param status_bus is a common notifier for tx statuses
+     * @param initial_timeout - streaming timeout when tx is not received
+     * @param nonfinal_timeout - streaming timeout when tx is being processed
+     */
+    CommandServiceTransportGrpc(
+        std::shared_ptr<CommandService> command_service,
+        std::shared_ptr<iroha::torii::StatusBus> status_bus,
+        std::chrono::milliseconds initial_timeout,
+        std::chrono::milliseconds nonfinal_timeout,
+        std::shared_ptr<shared_model::interface::TxStatusFactory>
+            status_factory);
+
+    /**
+     * Torii call via grpc
+     * @param context - call context (see grpc docs for details)
+     * @param request - transaction received
+     * @param response - no actual response (grpc stub for empty answer)
+     * @return - grpc::Status
```

Comment:
This line does not provide any useful information to the reader, same for methods below. Consider removing, or adding information about possible values and their cases.